io/net/bridge.py

```diff
@@ -61,6 +62,9 @@ def test_bridge_create(self):
         '''
         Set up the ethernet bridge configuration in the linux kernel
         '''
+        if os.path.exists('/etc/sysconfig/network-scripts/ifcfg-%s' % self.bridge_interface):
```

Comment:
@abdhaleegit Do we need to check all possible paths in supported distros ?

Reply:
Added the check for the same